packages/onboard-react/src/components/WalletDialog/WalletDialog.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC, useCallback, useMemo, useState } from "react";
+import {
+  Dialog,
+  VerticalTabs,
+  VerticalTabsItem,
+  Button,
+  ButtonStyle,
+  Heading,
+  DialogSize,
+} from "@sovryn/ui";
+import { FilterType, WalletList } from "../WalletList/WalletList";
+import { connectWallet$ } from "@sovryn/onboard-core/dist/streams";
+import { InstructionsTab } from "../InstructionsTab/InstructionsTab";
+import styles from "./WalletDialog.module.css";
+
+type WalletDialogProps = {
+  isOpen: boolean;
+};
+
+const WalletDialog: FC<WalletDialogProps> = ({ isOpen }) => {
+  const [index, setIndex] = useState(0);
+
+  const items: VerticalTabsItem[] = useMemo(
+    () => [
+      {
+        label: "Hardware Wallet",
+        infoText: "Select the hardware wallet you want to connect",
+        content: <WalletList filter={FilterType.hardware} />,
+        dataLayoutId: "hardware",
+      },
+      {
+        label: "Browser Wallet",
+        infoText: "Select the web3 wallet you want to connect",
+        content: <WalletList filter={FilterType.browser} />,
+        dataLayoutId: "browser",
+      },
+      {
+        label: "Don't have a wallet?",
+        infoText: "Read the following instructions",
+        content: <InstructionsTab />,
+        dataLayoutId: "instructions",
+      },
+    ],
+    []
+  );
+
+  const handleCloseClick = useCallback(() => {
+    connectWallet$.next({
+      inProgress: false,
+    });
+  }, []);
+
+  return (
+    <Dialog
```

Comment:
Corner radius on dialog is 8px instead of 4px

Reply:
Dialog also is not part of PR (and design i have it's 4px)

Reply:
Yep that's what I mean, you've got 8px at the moment but design shows 4px - Dialog component itself has correct size, so it seems to be applied via styling to VerticalTabs component from `packages\onboard-react\src\components\WalletDialog\WalletDialog.module.css`

Reply:
Got it, fixed